### DISCUSSION TITLE
Docs Feature Request: Request for a glossary

### DISCUSSION THREAD

**Discussion by juankennaugh** [algolia/instantsearch#5428] [Ideas]
It would be good if there was a glossary in the API to describe at least every method and property relating to state. there are so many terms: 'state, renderState, uiState, routeState, getWidgetUiState, getWidgetSearchParameters, getRenderState, getWidgetRenderState'.

Every time I go to the docs I end looking at loads of different pages to find one bit of information to cross reference to another page. Then obviously I forget most of it and have to do the same thing next time. Cheers

**Comment by Haroenv** [comment]
We recently wrote the summary of that page, but not yet the content. Good to know that it would help you!
